src/main/java/core/basesyntax/Plane.java

```diff
@@ -7,5 +7,78 @@
  */
 
 public class Plane {
+    private String name;
+    private String type;
+    private String color;
+    private double power;
+    private double speed;
 
+    public Plane(PlaneBuilder planeBuilder) {
```

Comment:
check https://mate-academy.github.io/jv-program-fulltime/01_core/checklist/hw_05.html about this one

Reply:
done